xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-server/xwiki-platform-search-solr-server-data/src/main/resources/xwiki/conf/schema.xml

```diff
@@ -840,12 +840,14 @@
     </fieldType>
 
     <!-- Czech -->
-    <fieldType name="text_cz" class="solr.TextField" positionIncrementGap="100">
+    <fieldType name="text_cs" class="solr.TextField" positionIncrementGap="100">
       <analyzer> 
         <tokenizer class="solr.StandardTokenizerFactory"/>
         <filter class="solr.LowerCaseFilterFactory"/>
         <filter class="solr.StopFilterFactory" ignoreCase="true" words="lang/stopwords_cz.txt" />
```

Comment:
Normally, I would agree as well, but since this means renaming the file too, it feels a bit wrong, as @mflorea pointed out before.

I`ve sent a mail on the solr list, asking about this confusion and hoping to get some clarifications: http://lucene.markmail.org/thread/fhgnyzeeaw3n2dul